README.md

```diff
@@ -1,2 +1,139 @@
 # Gotify Server
-[![Build Status](https://travis-ci.org/gotify/server.svg?branch=master)](https://travis-ci.org/gotify/server) [![codecov](https://codecov.io/gh/gotify/server/branch/master/graph/badge.svg)](https://codecov.io/gh/gotify/server) [![Go Report Card](https://goreportcard.com/badge/github.com/gotify/server)](https://goreportcard.com/report/github.com/gotify/server)
\ No newline at end of file
+[![Build Status][badge-travis]][travis] [![codecov][badge-codecov]][codecov] [![Go Report Card][badge-go-report]][go-report] [![Swagger Valid][badge-swagger]][swagger] [![Api Docs][badge-api-docs]][api-docs] [![latest release version][badge-release]][release]
+
+   * [Motivation](#motivation)
+   * [Features](#features)
+   * [Installation](#installation)
+     * [Docker](#docker)
+     * [Binary](#binary)
+   * [Configuration](#configuration)
+      * [File](#file)
+      * [Environment](#environment)
+      * [Database](#database)
+   * [Building](#building)
+      * [Cross-Platform](#cross-platform)
+   * [Tests](#tests)
+   * [Versioning](#versioning)
+   * [License](#license)
+
+## Motivation
+We wanted a simple server for sending and receiving messages (in real time per websocket). For this, not many open source projects existed and most of the existing ones were abandoned. Also, a requirement was, that it can be self-hostet, we know there are many free and commercial push services out there.
```

Comment:
Also, a requirement was that it can be self-hosted. We know there are many free and commercial push services out there.